Homework_3/lesson_3.6.py

```diff
@@ -0,0 +1,17 @@
+# Пользователь вводит строку текста. Вывести каждое слово с новой строки.
+# ✔ Строки нумеруются начиная с единицы.
+# ✔ Слова выводятся отсортированными согласно кодировки Unicode.
+# ✔ Текст выравнивается по правому краю так, чтобы у самого длинного
+# слова был один пробел между ним и номером строки.
+text = (input("Наберите любой текст: "))
+# строка для теста
+# text = "мой дядя самых честных правил когда не в шутку занемог"
+arr = text.split(" ")
+# поиск самого длинного слова
+length_of_the_most_longest_word = 0
+for i in arr:
+    if len(i) > length_of_the_most_longest_word:
+        length_of_the_most_longest_word = len(i)
```

Comment:
Блину самого блинного слова можно найти проще
```suggestion
length_of_the_most_longest_word = len(max(arr, key=len))
```